src/Questionnaire.css

```diff
@@ -1,8 +1,93 @@
-  .answerBtn {
-    position: relative;
-    padding: 8px;
-    margin: 4px;
-    font-size: 1em;
-    border-radius: 4px;
-    cursor: pointer;
-  }
\ No newline at end of file
+* {
+  /*all elements to inherit same font*/
+  font-family: "Montserrat", sans-serif;
+}
+
+.header h1 {
+  /*Questionnaire title*/
+  text-align: center;
+  margin: auto;
+  font-weight: bolder;
+  font-size: 50px;
+  color: #e0786c;
+}
+
+.questionnaireBox {
+  text-align: center;
+}
+
+input[type="radio"] {
+  /*make radio buttons bigger by scaling the size*/
+  transform: scale(2.5);
+  margin-right: 20px;
+}
+
+.header h2 {
+  /*let's get to know you*/
+  font-weight: bold;
+  font-size: 25px;
+  color: grey;
+}
+
+.header h4 {
+  /*please fill out the form*/
+  font-weight: bold;
+  font-size: 25px;
+  color: #4dadb7;
+}
+
+.boxed {
+  /*turqouise box section/background */
+  background-color: #4dadb7;
+  align-items: center;
+  margin-right: auto;
+  margin-left: auto;
+  width: 90%;
+  border-radius: 15px;
+  color: white;
+  padding: 6px;
+}
+
+.boxed p {
+  text-align: center;
+}
+
+.Questions {
+  font-size: 30px;
+  font-weight: bold;
+}
```

Comment:
Reduce weight of the answers. The bolded makes it difficult to read but the questions look good.